Add test plans for lecturer interface (#114)
Gamify-IT/docs: dev-manuals/test-plans/lecturer-interface-frontend/plans/u-lecturer-interface-03-show-specific-course.md

# Lecturer-Interface: Show specific course (`u.lecturer-interface-3`)

Version: V1.0, 08.08.2022 \
Author: Max Kästner

## Description

A specific course can be selected on the courses view to show up the details and edit it in the specific view.

## Precondition

At least one course is created.

## Postcondition

The details of a course are listed on a view.

## Typical procedure

1. Open the lecturer interface
2. The specific course is displayed on the courses view
3. Click on the row of the specific course
4. The site redirects to the specific course

> 5. The course details are show on the view like course name, description, semester, ...

Reviewer: ```suggestion
5. The course details are shown in the view (i.e. course name, description, semester, ...)
```

> 6. On the left side shows up a sidebar for navigation

Reviewer: ```suggestion
6. A sidebar for navigation shows up
```
